### PR TITLE
DESCW-534 Cosmetic patterns for CleanBC

### PR DESCRIPTION
**Pull request by codewisenate**
A series of CleanBC patterns with some related colour/gradients additions. Plus minor fixes identified by previous PR.

### CODE REVIEW

**Comment by codewisenate** [comment]
Non-cosmetic changes: removed CleanBC custom post types, fixed patterns that relied on them and cleaned up collapse block.

File: inc/core/theme-block-patterns.php [hunk 1]
```diff
@@ -124,19 +124,33 @@ function bcgov_blocks_theme_register_block_patterns() {
 		$block_patterns[] = 'cleanbc/cleanbc-home-cover-banner';
 		$block_patterns[] = 'cleanbc/cleanbc-info-cover-banner';
 		$block_patterns[] = 'cleanbc/cleanbc-landing-cover-banner';
+		$block_patterns[] = 'cleanbc/cleanbc-level-3-dark-banner';
```
Comment (ShawnTurple) [inline inc/core/theme-block-patterns.php:127]: these are a lot of patterns, with not so clear indication semantic wise what they are for., observation.. I know sometimes this is difficult, but just keep this in mind, when trying to fix a pattern a month from now.

File: src/Setup.php [hunk 2]
```diff
@@ -105,8 +86,8 @@ public function bcgov_block_theme(): void {
      */
     public function set_javascript_variables() {
 		$javascript_variables = array(
-			'domain'    => home_url(),
-			'site-name' => 'bcgov',
+			'domain'   => home_url(),
+			'siteName' => 'cleanbc',
```
Comment (ShawnTurple) [inline src/Setup.php]: not liking this being changed to cleanbc

**Review by ShawnTurple — APPROVED** [review]

File: inc/core/theme-block-patterns.php (hunk 1 above)
Comment (codewisenate) [reply inc/core/theme-block-patterns.php:127]: Patterns have a preview generated in the admin along with their title. This aids with identification and modification.

File: src/Setup.php (hunk 2 above)
Comment (codewisenate) [reply src/Setup.php]: It's not, that's a change that's coming out and will remain as bcgov in Setup.php

**Pull request merged**